Dockerfile

```diff
@@ -0,0 +1,39 @@
+# FROM ubuntu-upstart:precise
+# http://phusion.github.io/baseimage-docker/
+FROM phusion/baseimage:0.9.9
+
+# Install pre-pre-requisites.
+RUN apt-get update
```

Comment:
This isn't suitable for merge in any case, but best practice is to merge commands where it makes sense; it doesn't with update and install (IMO). Changing the command invalidates the intermediate container, but adding a new package shouldn't invalidate the `update`, usually.